src/main/java/teammates/sqllogic/core/FeedbackResponsesLogic.java

```diff
@@ -591,21 +591,25 @@ private SqlSessionResultsBundle buildResultsBundle(
         // build comment
         for (FeedbackResponseComment frc : allComments) {
             FeedbackResponse relatedResponse = frc.getFeedbackResponse();
-            FeedbackQuestion relatedQuestion = frc.getFeedbackQuestion();
             // the comment needs to be relevant to the question and response
-            if (relatedQuestion == null || relatedResponse == null) {
+            if (relatedResponse == null) {
+                continue;
+            }
+            FeedbackQuestion relatedQuestion = relatedResponse.getFeedbackQuestion();
```

Comment:
these guard clause and conditionally look checking the for FeedbackQuestion makes sense

Reply:
nice work on this